templates/home.html

```diff
@@ -29,11 +29,15 @@
 
         <div class="col center">
             <div class="in">
-                <h2>Shogun talks</h2>
+                <h2>Google Summer of Code 2015</h2>
                 <p align="justify">
-                    SHOGUN was presented in July at the <a href="https://ep2014.europython.eu/en/schedule/sessions/103/">Europython conference</a> in Berlin and in August at the <a href="http://hunch.net/~nyoml/">Open Machine Learning Workshop</a> in New York, both times by Heiko Strathmann. These talks are of interest both to people that do not know about SHOGUN and are interested in using it or just getting an introduction, as well as for people with prior experience using SHOGUN.
+                    <strong>We have applied as a mentoring org.<br></strong>
+                    Our list of project ideas this year is full of exciting and fun
+                    projects. There is also a new guide for prospective students. Let's now keep

```

Comment:
Maybe link to our GSoC 2015 page here?
Need to create it of course, and it should contain links to the wiki
@iglesias